src/task-1/Search/Search.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Search.css';
+import './../../styles/button.css';
+
+interface IProps {}
+
+const SearchInput = (props: { onChange: any }) => (
```

Comment:
Maybe try to keep components per file, even smallest

Reply:
ok will do in next tasks